### ISSUE TITLE
Mobile scrolling provides swiping animation

### ISSUE THREAD

**Issue by Thibaut-Brugidou** [sag1v/react-elastic-carousel#96]
**Describe the bug**
When you scroll the page on a mobile device touching the carousel, it will swipe the carousel in the same time.

**To Reproduce**
This bug is visible by simply resizing the window and scrolling the page (start touch must be onto the carousel)

**Expected behavior**
The carousel should understand this is a scroll and don't update his own position. An option canceling the real time swipe animation can also be a bypass solution.

Thanks a lot for this great work 👍

**Comment by sag1v** [comment]
Thanks @Thibaut-Brugidou !

**Comment by sag1v** [comment]
@Thibaut-Brugidou Can you please install a beta version and check if it works?

Either
```bash
npm i react-elastic-carousel@0.9.2-beta.2
```

or just use the `next` tag
```bash
npm i react-elastic-carousel@next
```

**Comment by Thibaut-Brugidou** [comment]
It's just perfect, thanks a lot ;)

**Comment by sag1v** [comment]
Released with version `0.9.3`
